### PR TITLE
release(v0.1.0-alpha.2): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.1.0-alpha.2 release.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/day-two/issues.
+
+### Contributors
+
+* Spencer Smith
+* Tim Jones
+* Tim Jones
```
Comment (TimJones) [inline CHANGELOG.md:14]: Still need to make this list unique contributors!

**Review by TimJones — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (smira) [reply CHANGELOG.md:14]: you changed your email ;) there's a way to merge contributors by specifying email aliases, but this is manual

**Pull request merged**